### ISSUE TITLE
[Calendar] Missing information about `min` and `max` for screen reader users

### ISSUE THREAD

**Issue by anna-lach** [sl-design-system/components#2990]
### Provide a general summary of the issue here

Currently screen reader user doesn't get any information when there is `min` and `max` set, so there is no information about the range of dates.

Please check [the example in storybook](https://storybook.sanomalearning.design/?path=/story/date-time-calendar--min-max).

Please check comment from Kasia: https://github.com/sl-design-system/components/issues/2568#issuecomment-3872080326

Maybe we could use [aria-valuemin](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-valuemin), but it is something to investigate.

### 👤 Your name

SLDS Team

### 🧢 Your Product/Team

SLDS
